index.html

```diff
@@ -86,23 +86,27 @@
 	</button>
 
 	<div id="navbar" class="theme-background-color theme-text-color windowDragHandle" tabindex="-1">
-		<button id="menu-button" class="fa fa-ellipsis-v navbar-action-button" data-label="openMenu"
-			tabindex="-1"></button>
-		<button id="back-button" hidden class="fa fa-angle-left navbar-action-button" data-label="goBack"
-			tabindex="-1"></button>
+		<button id="menu-button" class="navbar-action-button i carbon:overflow-menu-vertical" data-label="openMenu"
+			tabindex="-1">
+		</button>
+		<button id="back-button" hidden class="navbar-action-button i carbon:chevron-left" data-label="goBack"
+			tabindex="-1">
+		</button>
 		<div id="tabs">
 			<div id="tab-editor" hidden>
 				<input id="tab-editor-input" class="mousetrap" spellcheck="false"/>
 			</div>
 			<div id="tabs-inner" role="tablist"></div>
 		</div>
-		<button id="add-tab-button" class="fa fa-plus navbar-action-button" data-label="newTabAction"
-			tabindex="-1"></button>
-		<i class="fa fa-bars navbar-action-button invisible"></i>
+		<div class="navbar-right-actions">
+			<button id="add-tab-button" class="navbar-action-button i carbon:add" data-label="newTabAction" tabindex="-1">
+			</button>
+		</div>
 	</div>
 
-	<button id="switch-task-button" class="fa fa-bars navbar-action-button theme-text-color windowDragHandle"
-		data-label="viewTasks" tabindex="-1"></button>
+	<button id="switch-task-button" class="navbar-action-button theme-text-color windowDragHandle i carbon:chevron-down"
```

Comment:
I feel like a horizontal-menu button would be better, but the four-line one that Carbon has doesn't fit very well. I think I might try drawing one (like how the old reader button worked).

Reply:
Yeah, that's why I switched to chevrons. Guess I should leave it as-is for now?

Another options might be `carbon:overflow-menu-vertical`
![image](https://user-images.githubusercontent.com/11247099/88450643-f46bd780-ce82-11ea-8ae8-69fa4cca8ea8.png)

or `carbon:overflow-menu-horizontal`
![image](https://user-images.githubusercontent.com/11247099/88450661-14030000-ce83-11ea-989d-f41b4c970b40.png)

What do you think?

Reply:
`overflow-menu-vertical` is already used on the left side for the app menu on Windows and Linux; having two different buttons with the same icon would probably be too confusing. I also think it helps a bit to have the icon look like a "list", since it shows a vertical list when pressed.

I'll try making a three-bar menu SVG when I have some time tomorrow or Monday.